src/controllers/ratingsController.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-constant-condition */
+import model from '../database/models';
+import fService from '../services/facilityService';
+
+const addRating = async (req, res, next) => {
```

Comment:
The controller is the last request processing function in the chain, which next function are you going to call? The controller only takes two parameters; request and response